src/func/Microsoft.Health.Fhir.Ingest.Host/Microsoft.Health.Fhir.Ingest.Host.csproj

```diff
@@ -16,12 +16,12 @@
     <FhirVersion Condition="'$(FhirVersion)' == ''">R4</FhirVersion>
   </PropertyGroup>  
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.14.0" />
     <PackageReference Include="Microsoft.Azure.EventHubs.Processor" Version="4.1.0" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.EventHubs" Version="3.0.6" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.2" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="3.0.8" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" Version="3.0.14" />

```

Comment:
Curious, what is the different between these two packages?

Reply:
My understanding based on reading the code in the [webjobs git repo](https://github.com/Azure/azure-webjobs-sdk/tree/dev/src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights) is that the package Microsoft.Azure.WebJobs.Logging.ApplicationInsights is specific to Azure Functions and basically an extension of Microsoft.ApplicationInsights. The Microsoft.Azure.WebJobs.Logging.ApplicationInsights package contains a few additional things such as getting the Role, Host, etc of the function app (which we don't really need) but also includes some of the things we do need such as the adaptive sampling.